concepticondata/conceptlists/Oliveira-2014-517.tsv

```diff
@@ -0,0 +1,518 @@
+Oliveira-2014-517-229	229	areia	sand		671	SAND		§225
+Oliveira-2014-517-230	230	pedra	stone		857	STONE		§226
+Oliveira-2014-517-231	231	urucum (a árvore e o seu fruto)	achiote		349	ACHIOTE		§227
+Oliveira-2014-517-232	232	encima, cume	atop		57	ATOP		§228
+Oliveira-2014-517-233	233	cortar o cabelo	to cut hair		2155	CUT (WITH SCISSORS)		§229
+Oliveira-2014-517-234	234	chifre de animal	animal horn		1393	HORN (ANATOMY)		§230
+Oliveira-2014-517-235	235	golpear a cabeça	to hit oneself's head					§231
+Oliveira-2014-517-236	236	morrer	to die		1494	DIE		§232
+Oliveira-2014-517-237	237	esp. de formiga	species of ant					§233
+Oliveira-2014-517-238	238	torcer, girar, mover em círculos	to twist, to turn, to move in circles		3199	TWIST (AROUND)		§234
+Oliveira-2014-517-239	239	tocar, mexer com a mão, tatear	to touch, to stir with a hand, to grope		1892	TOUCH		§235
+Oliveira-2014-517-240	240	molhar, molhado	to wet, wet					§236
+Oliveira-2014-517-241	241	mão	hand		1277	HAND		§237
+Oliveira-2014-517-242	242	unha da mão	fingernail		1258	FINGERNAIL		§238
+Oliveira-2014-517-243	243	mão	hand		1277	HAND		§239
+Oliveira-2014-517-244	244	mão direita	right hand		2183	RIGHT HAND		§240
+Oliveira-2014-517-245	245	encontrar, procurar	to meet		1831	MEET		§241
+Oliveira-2014-517-246	246	engatinhar	to crawl		96	CRAWL		§242
+Oliveira-2014-517-247	247	dedo	finger		1303	FINGER		§243
+Oliveira-2014-517-248	248	barreiro, canamã	clay pit					§244
+Oliveira-2014-517-249	249	tu, você	you (sg.)		1215	THOU		§245
+Oliveira-2014-517-250	250	vocês, vós	you (pl.)		1213	YOU		§246a
+Oliveira-2014-517-251	251	pamonha	pamonha					§247
+Oliveira-2014-517-252	252	anzol	fishhook		1074	FISHHOOK		§248
+Oliveira-2014-517-253	253	veneno, amargo	poison, bitter				POISON//BITTER	§249
+Oliveira-2014-517-254	254	espinho	thorn		124	THORN		§250
+Oliveira-2014-517-255	255	ninho	nest		539	NEST		§251
+Oliveira-2014-517-256	256	esp. de preguiça (macaco)	species of monkey					§252
+Oliveira-2014-517-257	257	céu	sky		1732	SKY		§253
+Oliveira-2014-517-258	258	esp. de mosquito	species of mosquito					§254
+Oliveira-2014-517-259	259	cupim	termite		883	TERMITE		§255
+Oliveira-2014-517-260	260	soñar	dream		2374	DREAM		§256
+Oliveira-2014-517-261	261	abaixo, embaixo, sob	below, under		1485	BELOW OR UNDER		§257
+Oliveira-2014-517-262	262	carne	meat		634	MEAT		§258
+Oliveira-2014-517-263	263	jenipapo	genipa		3920	GENIPA		§259
+Oliveira-2014-517-264	264	colocar dentro, submergir	to put inside, to dive				INSERT//DIVE	§260
+Oliveira-2014-517-265	265	dentro, no centro	inside, in the center		1606	INSIDE		§261
+Oliveira-2014-517-266	266	banhar-se, tomar banho	to bathe, to take a bath		138	BATHE		§262
+Oliveira-2014-517-267	267	largo	wide		1243	WIDE		§263
+Oliveira-2014-517-268	268	morder	to bite		1403	BITE		§264
+Oliveira-2014-517-269	269	estrangeiro	stranger		791	STRANGER		§265
+Oliveira-2014-517-270	270	arco íris	rainbow		1733	RAINBOW		§266
+Oliveira-2014-517-271	271	capoeira, roça abandonada	abandoned rock					§267
```

Comment:
'abandoned field'; the Portuguese meanings talk more about a place/piece of land showing traces of past use